apps/journeys-admin/src/libs/useTextResponseWithButtonCreate/useTextResponseWithButtonCreate.spec.tsx

```diff
@@ -0,0 +1,104 @@
+import { InMemoryCache } from '@apollo/client'
+import { MockedProvider } from '@apollo/client/testing'
+import { act, renderHook, waitFor } from '@testing-library/react'
+
+import type {
+  BlockFields_ButtonBlock as ButtonBlock,
+  BlockFields_TextResponseBlock as TextResponseBlock
+} from '../../../__generated__/BlockFields'
+import {
+  ButtonColor,
+  ButtonSize,
+  ButtonVariant
+} from '../../../__generated__/globalTypes'
+
+import { useTextResponseWithButtonCreate } from './useTextResponseWithButtonCreate'
+import { textResponseWithButtonCreateMock } from './useTextResponseWithButtonCreate.mock'
+
+describe('useTextResponseWithButtonCreate', () => {
+  const blocks = {
+    textResponseBlock: {
+      id: 'textResponse.id',
+      parentBlockId: 'card.id',
+      parentOrder: 0,
+      label: 'Label',
+      hint: null,
+      minRows: null,
+      type: null,
+      routeId: null,
+      integrationId: null,
+      placeholder: null,
+      __typename: 'TextResponseBlock' as const
+    } as TextResponseBlock,
```

Comment:
```suggestion
    } 
```

What happens if you remove this?

Reply:
From my understanding `as TextResponseBlock` allow you to add extra properties that don't belong to the object and forces it to be treated as a TextResponseBlock.

Reply:
Probably using something like `satisfies TextResponseBlock` if you want this to be type safe